src/HomeArticles.jsx

```diff
@@ -0,0 +1,19 @@
+import { allArticles } from './Api';
+import { useEffect } from 'react';
+import { SingleArticle } from './SingleArticle';
+
+export const HomeArticles = ({ articles, setArticles }) => {
+	useEffect(() => {
+		allArticles().then((response) => {
+			setArticles(response.articles);
+		});
+	}, [setArticles]);
+
+	return (
+		<ul>
+			{articles.map((article) => {
+				return <SingleArticle key={article.article_id} article={article} />;
```

Comment:
This naming here could get really confusing when you come to create a single Article page. I would use something like "ArticleCard" here.